app/pages/lab/workflow-create-dialog.cjsx

```diff
@@ -0,0 +1,58 @@
+React = require 'react'
+ReactDOM = require 'react-dom'
+counterpart = require 'counterpart'
+
+WorkflowCreateDialog = React.createClass
```

Comment:
This name is a little confusing; it's not a dialog, it's a form that gets put into a dialog elsewhere.